spec/index.bs

```diff
@@ -9532,38 +9533,51 @@ None of the depth formats can be filtered.
         <td>1 &minus; 4
         <td>stencil
         <td>{{GPUTextureSampleType/"uint"}}
+        <td>`vec4<u32>(S, S, S, S)` or `vec4<u32>(S, 0, 0, 1)`
```

Comment:
can also write `vec4<u32>(S)`

Reply:
This needs to be S,X,X,X

Reply:
I don't think this matches the prose below.